workbench/config/WorkbenchConfig.php

```diff
@@ -52,33 +52,35 @@ function __construct() {
         // load environment variable based overrides
         $configNamespace = "forceworkbench";
         $configDelim = "__";
+
```

Comment:
Could you clean up this file so that the diff looks cleaner? There are too many whitespace changes.